hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/span/SpanMapManager.java

```diff
@@ -328,12 +329,16 @@ private void expandImpliedTransfers(final TxnAccessor accessor) {
     }
 
     public static void reCalculateXferMeta(final TxnAccessor accessor, final ImpliedTransfers impliedTransfers) {
+        final var maybeAssessedCustomFees = impliedTransfers.getAssessedCustomFeeWrappers();
+        if (maybeAssessedCustomFees.isEmpty()) {
+            return;
+        }
```

Comment:
Is this needed ?

Reply:
Not needed for this fix, just a tiny optimization that happened to have been done post-`release/0.47` being cut.

Reply:
Ahh removed it anyways, unit tests were failing.